homeassistant/components/random/strings.json

```diff
@@ -83,6 +83,7 @@
     "sensor_device_class": {
       "options": {
         "apparent_power": "[%key:component::sensor::entity_component::apparent_power::name%]",
+        "area": "[%key:component::sensor::entity_component::area::name%]",
```

Comment:
Integrations such as this / scrape etc. should not be included in this PR. That can be fixed in follow-ups.

Reply:
I've removed from here and scrape, are you also meaning number, sensor and recorder?